packages/frontend/src/redux/crossStateSelectors/selectCollectedAvailableForClaimData.js

```diff
@@ -70,3 +70,36 @@ export default createSelector(
         });
     }
 );
+
+export const selectCollectedAvailableForClaimDataForAccountId = createSelector(
```

Comment:
```suggestion
export const selectCollectedAvailableForClaimDataByAccountId = createSelector(
```
This is a bit of a mouthful 😅  - I feel like this makes it a bit clearer, thoughts?

Reply:
😆 yeah its less repetitive